sdk/lib/rtl/actctx.c

```diff
@@ -23,6 +23,21 @@
 
 #include <wine/unicode.h>
 
+#define GetProcessHeap() RtlGetProcessHeap()
+#define GetCurrentProcess() NtCurrentProcess()
+#define FIXME DPRINT1
+#define WARN DPRINT1
+#define TRACE DPRINT
+#define __TRY _SEH2_TRY
+#define __EXCEPT_PAGE_FAULT _SEH2_EXCEPT(EXCEPTION_EXECUTE_HANDLER)
+#define __ENDTRY _SEH2_END
+#define FILE_END_OF_FILE_INFORMATION FILE_STANDARD_INFORMATION
+#define FileEndOfFileInformation FileStandardInformation
```

Comment:
hmmm why is this needed? (FileEndOfFileInformation is an info class that actually exists)

Reply:
This is from https://github.com/reactos/reactos/commit/e2e8b52413e940619675278fe98b9a93ed0eb221
```
- Modify get_manifest_in_manifest_file function to use FileStandardInformation vice FileEndOfFileInformation (currently unsupported) in call to NtQueryInformationFile.
```
And yes, it's still unimplemented.
Looking into it.

Reply:
Turns out this isn't implemented on Windows either, at least not in user mode.

Reply:
Ah yes indeed, and I noticed that fact too a long time ago:
https://git.reactos.org/?p=reactos.git;a=blob;f=base/services/eventlog/file.c;hb=91acf823d8d4e2b6b6e8012c98b1f178922b1547#l357

Reply:
Because every line of hack in the file makes syncing harder. If there is only one block of our code at the top, it much less of a problem as if every 3rd line has some hack in it, like it was before, which makes syncing impossible.